include/odp/api/spec/ipsec.h

```diff
@@ -762,6 +768,38 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * IPSEC stats content
+ */
+typedef struct odp_ipsec_stats_t {
+	/** Number of packets processed successfully */
+	uint64_t num_success;
+
+	/** Number of packets with protocol errors */
+	uint64_t num_proto_err;
+
+	/** Number of packets with SA lookup failures */
+	uint64_t num_sa_lookup_err;
+
+	/** Number of packets with authentication errors */
+	uint64_t num_auth_err;
+
+	/** Number of packets with antireplay check failures */
+	uint64_t num_antireplay_err;
+
+	/** Number of packets with algorithm errors */
+	uint64_t num_alg_err;
+
+	/** Number of packes with MTU errors */
+	uint64_t num_mtu_err;
+
+	/** Number of packets with hard lifetime(bytes) expired */
+	uint64_t num_hard_exp_bytes_err;
+
+	/** Number of packets with hard lifetime(packets) expired */
+	uint64_t num_hard_exp_pkts_err;
```

Comment:
num_ prefix is not needed for stat counters

Reply:
Agreed. Will make this change with next version.